apps/backoffice/src/providers/app.tsx

```diff
@@ -17,20 +19,31 @@ interface AppProviderProps {
   children: ReactNode;
 }
 
-export const AppProvider = ({ children }: AppProviderProps) => (
-  <ThemeProvider theme={appTheme}>
-    <CssBaseline />
-    <DialogProvider>
-      <DrawerProvider>
-        <SnackbarProvider
-          Components={{
-            custom: Notification,
-          }}
-          maxSnack={5}
-        >
-          {children}
-        </SnackbarProvider>
-      </DrawerProvider>
-    </DialogProvider>
-  </ThemeProvider>
-);
+export const AppProvider = ({ children }: AppProviderProps) => {
+  //Mixpanel Initialization
+  mixpanel.init(getConfiguration().BACK_OFFICE_MIXPANEL_TOKEN, {
+    debug: true,
+    verbose: true,
+    track_pageview: false,
```

Comment:
Create a function on mix-panel.ts dedicated utility file and then call it here.

Reply:
[Done](https://github.com/pagopa/io-services-cms/pull/1018/commits/abc169ab602543d7a6658e84e1eba7c5ddafab6a)

Reply:
ok, clear old commented code 😉